packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -332,7 +332,7 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
       _scrollOffset = newScrollOffset;
     });
     PageStorage.of(context)?.writeState(context, _scrollOffset);
-    new ScrollNotification(this, _scrollOffset).dispatch(context);
+    new ScrollNotification(this, ScrollNotificationKind.updated).dispatch(context);
```

Comment:
Should we move this into `dispatchOnScroll` for symmetry?  As written, you might get an update before a start.